### ISSUE TITLE
Accessibility: missing aria-label for breadcrumbs

### ISSUE THREAD

**Issue by josecervera** [SAP/spartacus#3608]
As per title, the aria-label is missing in the breadcrumb

**Comment by developpeurweb** [comment]
ARIA attributes are not under the scope of the first accessibility release, therefor removing from `A11Y-R1` and setting it for the next round.

**Comment by developpeurweb** [comment]
After some research and testing this one below turned out to be a good cross-SR combination, please ping me when ready to test.

<img width="783" alt="Screen Shot 2021-06-28 at 1 09 48 PM" src="https://user-images.githubusercontent.com/15113219/123678478-57dd5280-d814-11eb-872e-9244d4efd836.png">

**Comment by KateChuen** [comment]
Tested on develop branch.

OK now, breadcrumbs has aria-label
<img width="1188" alt="Screen Shot 2022-05-18 at 3 47 13 PM" src="https://user-images.githubusercontent.com/45409515/169144019-632e17f8-5521-462a-8468-5c245c764dd4.png">

![breadcrumbs](https://user-images.githubusercontent.com/45409515/169144052-1b3515fd-dc5c-48fa-b854-943404640293.png)
